scss/components/_activity-month-nav.scss

```diff
@@ -0,0 +1,37 @@
+.civicase__activity-month-nav__group {
+  border-left: 2px solid $activity-month-nav-divider;
+
+  [class^='civicase__activity-month-nav__group-'] {
```

Comment:
Any specific reason to add padding to the children div's and not to the parent div. I think if we  apply padding-left of 15 px to the parent `.civicase__activity-month-nav__group` it will have the same effect. Please let me know if its not the case